components/gitpod-protocol/src/protocol.ts

```diff
@@ -1200,6 +1200,16 @@ export namespace AdditionalContentContext {
     }
 }
 
+export interface OpenPrebuildContext extends WorkspaceContext {
```

Comment:
While I'm not super-fond of adding a ContextParser for this I understand that currently, it's the only accessible way that make things like this work.
Also, removing it later should not be too hard. :slightly_smiling_face:

Reply:
Why don't we use the commit context?

Reply:
See related discussion: https://github.com/gitpod-io/gitpod/pull/13706#discussion_r991574427